spec/models/item_spec.rb

```diff
@@ -43,4 +39,27 @@
   xit 'responds to orders' do
     expect(item).to respond_to(:orders)
   end
+
+  it 'must have a unique title' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.